components/home/sessoes/Visao.vue

```diff
@@ -9,4 +22,38 @@ export default {
 }
 </script>
 <style lang="scss" scoped>
+.sessao {
+  margin: 40px 0 40px 0px;
+}
+ul {
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  margin: auto;
+  max-width: 1100px;
+}
+li {
+  margin: 0 10px 0 10px;
+  padding: 20px 0 15px 40px;
+  text-decoration: none;
+  list-style: none;
+  background-repeat: no-repeat;
+  background-position: left center;
+  background-size: 24px;
+  font-size: 1.4rem;
+  color: #5a6166;
+  background-image: url('~static/hexagon.svg');
```

Comment:
Pelos padrões, acho que não é apropriado `hexagon.svg` em static, mas sim em *assets*. 

```suggestion
  background-image: url('~assets/hexagon.svg');
```

Reply:
Por que o svg fica em assets e o png fica em static? não entendi @thayannevls

Reply:
Não tem a ver com o tipo de arquivo, é mais por padrão de onde colocar as coisas mesmo https://nuxtjs.org/guide/directory-structure